examples/rollbar-spring-boot-webmvc/src/main/java/com/example/springbootwebmvc/RollbarConfig.java

```diff
@@ -0,0 +1,45 @@
+package com.example.springbootwebmvc;
+
+import com.rollbar.notifier.Rollbar;
+import com.rollbar.notifier.config.Config;
+import com.rollbar.spring.webmvc.RollbarSpringConfigBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+
+@Configuration()
+@EnableWebMvc
+@ComponentScan({
+        "com.example.springbootwebmvc",
+        "com.rollbar.spring.boot.webmvc",
```

Comment:
IMHO I think it's just clearer to specify as a package to scan `com.rollbar.spring` and the client's package of the integration, that in this case is this example project. I'd opt for this:

```
@ComponentScan({
        "com.example.springbootwebmvc",
        "com.rollbar.spring"
})
```

As this level of detail does not provide any value and may cause some confusion.